Tests/IGListAdapterTests.m

```diff
@@ -27,6 +27,12 @@
 XCTAssertEqual(CGPointEqualToPoint(point, p), YES); \
 } while(0)
 
+#define IGAssertEqualSize(size, w, h, ...) \
+do { \
+CGSize s = CGSizeMake(w, h); \
+XCTAssertEqual(CGSizeEqualToSize(size, s), YES); \
+} while(0)
+
```

Comment:
🤔  hm... i think i'd rather remove this (especially since it's copied multiple places)

instead, `XCTAssertTrue(CGSizeEqualToSize(....));`

thoughts? @rnystrom ?